server/src/main/kotlin/com/kamelia/jellyfish/rest/user/Routes.kt

```diff
@@ -9,9 +10,13 @@ import io.ktor.server.application.call
 import io.ktor.server.routing.Route
 import io.ktor.server.routing.route
 
+const val pathUUID = "/{uuid}"
```

Comment:
Did you extract this string in a `val` because of Sonar Linter? Because, since this is not a magic string, I think this advice of the linter should be ignored.